.editorconfig

```diff
@@ -0,0 +1,15 @@
+# https://editorconfig.org/
```

Comment:
@jorizci doesn't like editorconfig, but I find it useful. You can include it or leave it out.